ios/Podfile

```diff
@@ -43,11 +63,21 @@ target 'gutenberg' do
   pod 'Gutenberg', :path => '../Gutenberg.podspec'
 
   target 'gutenbergTests' do
-    inherit! :search_paths
+    inherit! :complete
```

Comment:
@SergioEstevao I have saw this change when I used this tool: https://react-native-community.github.io/upgrade-helper/?from=0.61.5&to=0.62.0
not sure if we need it or not, please re-check if you can, thanks!